### ISSUE TITLE
Sampling

### ISSUE THREAD

**Issue by whyzjuhit** [vinecopulib/rvinecopulib#269]
Hi,
I'm using rvine fuction to generate samples. And I find that the samples are generated radomly. But before sampling, actually I have some data already(sampled by other method), so they are fixed. I want to based on these data and then use vine method to generated the other correlated variables. I think I need to do a rosenblatt tranformation or something. I guess: 
1. I can use vine function fit the observations. 
2. I can use rosenblatt fuction to convert the data generated by other method to pseudo_obs
3. based on the fitted vine and the pseudo_obs, I can use inverse_rosenblatt funtion to sample
Is it right? What else should I do?

**Comment by whyzjuhit** [comment]
Yes, I want to do a conditional sampling. These days I tried with rvinecopulib. I found the pseudo_obs of a matrix x is different from what rosenblatt fuction gives. It seems I can't solve it with rvinecopulib.

**Comment by tnagler** [comment]
Indeed the pseudo_obs and rosenblatt functions do different things. The first converts to uniform variables without changing the dependence, the second to independent uniform variables. For conditional sampling you need the second.

**Comment by whyzjuhit** [comment]
Thank you Professor Nagler. I've read the paper and found it difficult for me. I'll keep on working on it. Thank you again.

**Comment by tnagler** [comment]
The values are really arbitrary and should not influence the transform of the other variables if the structure is set up correctly. In particular,  the variables you condition on have to be on the top right of the diagonal. As a sanity check you can use `NA` as the 'arbitrary value' and see if the transform comes out with `NA`s at the wrong place.

**Comment by HEY745** [comment]
Sorry to bother you again, Professor Nagler. Could you clarify about"the variables you condition on have to be on the top right of the diagonal"? I try to understand it from two aspects. On the one hand, variable data should be placed on the top right of the diagonal(step 2), but according to the article, variable data should be placed like that.
![image](https://github.com/vinecopulib/rvinecopulib/assets/144755320/f50605eb-c618-45a7-9714-c370bcbd262a)
On the other hand,  it may the matrix structure of Vine Copula. But I found that the Vine Copula  is the same whether it is the upper or lower triangle.
![image](https://github.com/vinecopulib/rvinecopulib/assets/144755320/d0ed3e0f-66b7-41d3-84af-bb3bc2c0dba2)

**Comment by tnagler** [comment]
This is a different package (VineCopula instead of rvinecopulib) which uses a different convention for the structure matrix.  Here, the variables you condition on have to be bottom right on the diagonal. Please ask questions about VineCopula in the respective repository.
